ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,386 @@
+                                                fieldDescription="Display the fashion size text"
+                                                fieldLabel="Fashion Size"
+                                                name="./fashionSize"/>
+                                            <productSize
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                                                composite="{Boolean}true">
+                                                <field
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/container"
+                                                    name="./productSize">
+                                                    <items jcr:primaryType="nt:unstructured">
+                                                        <selectSize
+                                                            jcr:primaryType="nt:unstructured"
+                                                            sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                            name="./size"/>
+                                                    </items>
+                                                </field>
+                                            </productSize>
+											 <selectedFashionSize
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="Display the selected fashion size text"
+                                                fieldLabel="Selected Fashion Size"
+                                                name="./selectedFashionSize"/>
+                                            <Quantity
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="Display quantity text"
+                                                fieldLabel="Quantity"
+                                                name="./quantity"/>
+					    <noProductAvailable
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="Display no product avilable"
+                                                fieldLabel="No Product avilable"
```

Comment:
@Rajamouli-hcl  please add Label in fieldLabel

Reply:
added label